tox.ini

```diff
@@ -366,7 +366,9 @@ commands =
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-cosmos
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-fetchai
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-cli-ipfs
-    pytest -rfE --doctest-modules aea packages/fetchai/connections packages/fetchai/protocols packages/fetchai/skills tests/ --cov=aea --cov=packages/fetchai/connections --cov=packages/fetchai/contracts --cov=packages/fetchai/protocols --cov=packages/fetchai/skills --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc {posargs}
+
+    aea test packages {posargs}
+    pytest -rfE --doctest-modules aea tests/ --cov=aea --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc --cov-append {posargs}
```

Comment:
I think we can remove the coverage check from this, since we only update the coverage check from ubuntu-py3.10 run. (this might also speed up the CI)